src/jgikbase/test/idmapping/mongo_controller.py

```diff
@@ -0,0 +1,60 @@
+from pathlib import Path
```

Comment:
More documentation in this file would be v helpful

Reply:
Done